filters/lang/filter.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from langid.langid import LanguageIdentifier, model
+
+
+class LanguageFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ['af', 'am', 'an', 'ar', 'as', 'az', 'be', 'bg', 'bn', 'br', 'bs', 'ca', 'cs', 'cy', 'da', 'de',
+                 'dz', 'el', 'en', 'eo', 'es', 'et', 'eu', 'fa', 'fi', 'fo', 'fr', 'ga', 'gl', 'gu', 'he', 'hi',
```

Comment:
Also, please add the [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) variable here.